sprout/sprout/settings.py

```diff
@@ -158,6 +158,19 @@
     }
 }
 
+CLEANUP_RULES = [
+    # appliance name , max appliance lifetime
+    {'name': r'^test', 'lifetime': dict(hours=1)},
```

Comment:
I'm thinking these don't need to be in this cleanup, since they aren't related to any machines that sprout deploys. The intent of these cleanup strings in our current VM cleanup is for test resources, which I don't think sprout should be trying to cleanup.

We shouldn't be creating test resources on sprout providers - and if we do (for whatever reason) I think I would like to keep cleanup of VM's named this way to the cleanup_old_vms script, which can be easily run against sprout providers.

Thoughts?

Reply:
I guess you're right. Off the top of my head, appliances which are used in tests aren't renamed to test-*